book/src/faqV2.md

```diff
@@ -0,0 +1,487 @@
+
+After checkpoint forwards sync completes, the beacon node will start to download historical blocks. The log will look like:
+
+```bash
+INFO Downloading historical blocks           est_time: --, distance: 4524545 slots (89 weeks 5 days), service: slot_notifier
+```
+
+If the same log appears every minute and you do not see progress in downloading historical blocks, you can try one of the followings:
+ 
+   - Check the number of peers you are connected to. If you have low peers (less than 50), try to do port forwarding on the port 9000 TCP/UDP to increase peer count.
+   - Restart the beacon node.
+
+
+### I proposed a block but the beacon node shows `could not publish message` with error `duplicate` as below, should I be worried?
+
+```
+INFO Block from HTTP API already known`
+WARN Could not publish message error: Duplicate, service: libp2p
+```
+
+This error usually happens when users are running mev-boost. The relay will publish the block on the network before returning it back to you. After the relay published the block on the network, it will propagate through nodes, and it happens quite often that your node will receive the block from your connected peers via gossip first, before getting the block from the relay, hence the message `duplicate`. 
+
+In short, it is nothing to worry about.
+
+### I see beacon node logs `Head is optimistic`, and I am missing attestations. What should I do?
+
+The log looks like:
+
+```
+WARN Head is optimistic       execution_block_hash: 0x47e7555f1d4215d1ad409b1ac188b008fcb286ed8f38d3a5e8078a0af6cbd6e1, info: chain not fully verified, block and attestation production disabled until execution engine syncs, service: slot_notifier
+```
+
+It means the beacon node will follow the chain, but it will not be able to attest or produce blocks. This is because the execution client is not synced, so the beacon chain cannot verify the authenticity of the chain head, hence the word `optimistic`. What you need to do is to make sure that the execution client is up and syncing. Once the execution client is synced, the error will disappear.
+
+### My beacon node logs `CRIT Beacon block processing error error: ValidatorPubkeyCacheLockTimeout, service: beacon`, what should I do? 
+
+An example of the log is shown below:
+
+```
+CRIT Beacon block processing error           error: ValidatorPubkeyCacheLockTimeout, service: beacon
+WARN BlockProcessingFailure                  outcome: ValidatorPubkeyCacheLockTimeout, msg: unexpected condition in processing block.
+```
+
+A `Timeout` error suggests that the computer may be overloading at the moment, for example, the execution client is still syncing. You may use the flag `--disable-lock-timeouts` to silence this error, although it will not fix the underlying slowness. Nevertheless, this is a relatively harmless log, and the error should go away once the resources used are back to normal. 
```

Comment:
```suggestion
A `Timeout` error suggests that the computer may be overloaded at the moment, for example, the execution client is still syncing. You may use the flag `--disable-lock-timeouts` to silence this error, although it will not fix the underlying slowness. Nevertheless, this is a relatively harmless log, and the error should go away once the resources used are back to normal. 
```